src/renderer.ts

```diff
@@ -231,7 +231,9 @@ const textElement = (format: Format, supportsDarkMode = true) => (
 						children,
 				  );
 		case 'BLOCKQUOTE':
-			return h(Blockquote, { key, format }, children);
+			return !isEditions
+				? h(Blockquote, { key, format }, children)
+				: h('blockquote', { key }, children);
```

Comment:
Nitpick. Might be preferable to swap the order to avoid the `!`, i.e.:

```ts
isEditions ? h('blockquote', ...) : h(Blockquote, ...);
```

Just slightly less cognitive load for the reader.

Reply:
Good shout, fixed.